### PR TITLE
fixed tests and added sentiment_identifier.py

### PR DESCRIPTION
**Pull request by ashwoodaa**

### CODE REVIEW

**Comment by keithbradley1** [comment]
good stuff

**Comment by keithbradley1** [comment]
👍

File: sentiment_lexicon.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
-sentiment_lexicon =
-  {
-    "positive": [("totally": .5), ("good": .5), ("smart": .5)],
-    "negative": [("divided": .5), ("not": 1, ("tough": .5)],
-    "neutral": [("country": .5), ("watching": .5), ("looking": .5)]
+sentiment_lexicon = {
+  "positive": [('totally': .5), ('good': .5), ('smart': .5)],
```
Comment (carter-harris) [inline sentiment_lexicon.py]: commas and no semi dudes

File: sentiment_lexicon.py [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
+  "negative": [('divided': .5), ('not': 1, ('tough': .5)],
```
Comment (carter-harris) [inline sentiment_lexicon.py]: missing a ")" in the second tuple

File: sentiment_lexicon.py [hunk 3]
```diff
@@ -1,8 +1,7 @@
...
+  "neutral": [('country': .5), ('watching': .5), ('looking': .5)]
```
Comment (Jessicashinjo) [inline sentiment_lexicon.py]: Inside the tuples go ahead and make the separators commas instead of colons. Tuples use commas to separate items.

**Comment by Jessicashinjo** [comment]
👍

File: test_sentiment_analysis.py [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
+from sentiment_identifier import *
+
+class TestSentimentIdentifier(unittest.TestCase):
   ''' Receives strings to be searched in the dictionary '''
 
   @classmethod
   def setUpClass(self):
-    self.domain = Domain_Identifier()
+    self.sentiment = Sentiment_Identifier()
     pass
```
Comment (keithbradley1) [inline test_sentiment_analysis.py]: looks like you don't need 'pass' here

**Comment by keithbradley1** [comment]
current version looks good to merge

**Pull request merged**